c/src/measureresult.cpp

```diff
@@ -38,33 +38,52 @@ tirexError tirexResultEntryNum(const tirexResult* result, size_t* num) {
 
 void tirexResultFree(tirexResult* result) { delete result; }
 
-template <class... Ts>
-struct overloaded : Ts... {
-	using Ts::operator()...;
-};
+static std::string toYAML(const tirex::TimeSeries<unsigned>& timeseries) {
+	const auto& [timestamps, values] = timeseries.timeseries();
+
+	std::stringstream timestampsStream;
+	timestampsStream << "[";
+	bool firstTImestamp = true;
+	for (auto& elem : timestamps) {
+		if (!firstTImestamp)
+			timestampsStream << ",";
+		timestampsStream << "\"";
+		timestampsStream << elem.count();
+		timestampsStream << "\"";
+		firstTImestamp = false;
+	}
+	timestampsStream << "]";
 
-template <class... Ts>
-overloaded(Ts...) -> overloaded<Ts...>;
+	std::stringstream valuesStream;
+	valuesStream << "[";
+	bool firstValue = true;
+	for (auto& elem : values) {
+		if (!firstValue)
+			valuesStream << ",";
+		valuesStream << "\"";
+		valuesStream << elem;
+		valuesStream << "\"";
+		firstValue = false;
+	}
+	valuesStream << "]";
```

Comment:
Should be split into its own function that is called once for timestamps and once for values.